chaoscenter/web/src/views/AccountSettingsUserManagement/AccountSettingsUserManagement.tsx

```diff
@@ -190,6 +191,13 @@ export default function AccountSettingsUserManagementView(
   const { isOpen: isCreateUserModalOpen, open: openCreateUserModal, close: closeCreateUserModal } = useToggleOpen();
   const { getString } = useStrings();
 
+  const [currentPage, setCurrentPage] = useState(1);
+  const itemsPerPage = 5;
```

Comment:
Users can manually set limits too, you can do something like `src/controllers/ExperimentDashboardV2/ExperimentDashboardV2.tsx` what done here.

Reply:
Actually I have my minors this week, would continue from next week. Apologies for delay.

Reply:
Sure no problems @kartikaysaxena, all the best for your exams.

Reply:
I have added options for the user to select the number of itemsPerPage. Should I add more options or change it into an input field or does it look fine? PTAL @hrishavjha 

https://github.com/litmuschaos/litmus/assets/120778728/2cf4b66d-40bb-47ec-a1a7-6bb811a99812

Reply:
If you pass `pageSizeOptions: [...new Set([15, 30, limit])].sort()` prop to pagination component it will render the dropdown. Rest changes looks good to me 🚀